src/Sarif.UnitTests/TestData/PrereleaseCompatibilityTransformer/Inputs/WithExternalPropertyFiles.sarif

```diff
@@ -0,0 +1,56 @@
+﻿{
+    "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2018-10-10",
+    "version": "2.0.0-csd.2.beta.2018-10-10",
+    "runs": [{
+      "tool": {
+        "name": "CodeScanner",
+        "semanticVersion": "2.1.0"
+      },
+      "results": [],
+      "externalPropertyFiles" : {
+        "conversion": {
+            "artifactLocation": {
+                "uri": "Test/Test.txt",
```

Comment:
>Test/Test.txt [](start = 24, length = 13)

name this 'externalized/conversion.sarif-external-properties'. no uriBaseId. remove itemCount for anything that's not an array